sql/src/test/scala/hydra/sql/JdbcRecordWriterSpec.scala

```diff
@@ -508,5 +509,51 @@ class JdbcRecordWriterSpec extends Matchers
         rs.next() shouldBe false
       }.get
     }
+
+    it ("throws a BatchUpdateException when null bytes exist in a string") {
+      val configPG = ConfigFactory.parseString(
+        """
+          |connection.url = "jdbc:postgresql://localhost:5432/postgres"
+          |connection.user = postgres
+          |
+      """.stripMargin)
+
+      val providerPG = new DriverManagerConnectionProvider("jdbc:postgresql://localhost:5432/postgres",
```

Comment:
use configPG here to get the host/port/db combo instead of the hard coded string?